### ISSUE TITLE
pylint 2.4.1 breaks the build

### ISSUE THREAD

**Issue by gerboengels** [mystor/git-revise#41]
pylint 2.3.1 works OK. pylint is updated to 2.4.1, and gives issues

```
py36 run-test: commands[1] | pylint gitrevise
************* Module gitrevise.odb
gitrevise/odb.py:141:8: E0242: Value 'workdir' in slots conflicts with class variable (class-variable-slots-conflict)
gitrevise/odb.py:142:8: E0242: Value 'gitdir' in slots conflicts with class variable (class-variable-slots-conflict)
gitrevise/odb.py:143:8: E0242: Value 'default_author' in slots conflicts with class variable (class-variable-slots-conflict)
gitrevise/odb.py:144:8: E0242: Value 'default_committer' in slots conflicts with class variable (class-variable-slots-conflict)
gitrevise/odb.py:145:8: E0242: Value 'index' in slots conflicts with class variable (class-variable-slots-conflict)
gitrevise/odb.py:146:8: E0242: Value '_objects' in slots conflicts with class variable (class-variable-slots-conflict)
gitrevise/odb.py:147:8: E0242: Value '_catfile' in slots conflicts with class variable (class-variable-slots-conflict)
gitrevise/odb.py:148:8: E0242: Value '_tempdir' in slots conflicts with class variable (class-variable-slots-conflict)
gitrevise/odb.py:400:17: E0242: Value 'repo' in slots conflicts with class variable (class-variable-slots-conflict)
gitrevise/odb.py:400:25: E0242: Value 'body' in slots conflicts with class variable (class-variable-slots-conflict)
gitrevise/odb.py:400:33: E0242: Value 'oid' in slots conflicts with class variable (class-variable-slots-conflict)
gitrevise/odb.py:400:40: E0242: Value 'persisted' in slots conflicts with class variable (class-variable-slots-conflict)
gitrevise/odb.py:471:17: E0242: Value 'tree_oid' in slots conflicts with class variable (class-variable-slots-conflict)
gitrevise/odb.py:471:29: E0242: Value 'parent_oids' in slots conflicts with class variable (class-variable-slots-conflict)
gitrevise/odb.py:471:44: E0242: Value 'author' in slots conflicts with class variable (class-variable-slots-conflict)
gitrevise/odb.py:471:54: E0242: Value 'committer' in slots conflicts with class variable (class-variable-slots-conflict)
gitrevise/odb.py:471:67: E0242: Value 'message' in slots conflicts with class variable (class-variable-slots-conflict)
gitrevise/odb.py:475:29: E1101: Instance of 'Commit' has no 'body' member (no-member)
gitrevise/odb.py:496:15: E1101: Instance of 'Commit' has no 'repo' member (no-member)
gitrevise/odb.py:500:16: E1101: Instance of 'Commit' has no 'repo' member (no-member)
gitrevise/odb.py:507:39: E1101: Instance of 'Commit' has no 'oid' member (no-member)
gitrevise/odb.py:549:15: E1101: Instance of 'Commit' has no 'repo' member (no-member)
gitrevise/odb.py:558:23: E1101: Instance of 'Commit' has no 'oid' member (no-member)
gitrevise/odb.py:601:17: E0242: Value 'repo' in slots conflicts with class variable (class-variable-slots-conflict)
gitrevise/odb.py:601:25: E0242: Value 'mode' in slots conflicts with class variable (class-variable-slots-conflict)
gitrevise/odb.py:601:33: E0242: Value 'oid' in slots conflicts with class variable (class-variable-slots-conflict)
gitrevise/odb.py:646:17: E0242: Value 'entries' in slots conflicts with class variable (class-variable-slots-conflict)
gitrevise/odb.py:650:15: E1101: Instance of 'Tree' has no 'body' member (no-member)
gitrevise/odb.py:656:39: E1101: Instance of 'Tree' has no 'repo' member (no-member)
gitrevise/odb.py:666:22: E1101: Instance of 'Tree' has no 'repo' member (no-member)
gitrevise/odb.py:667:8: E1101: Instance of 'Tree' has no 'repo' member (no-member)
gitrevise/odb.py:678:24: E1101: Instance of 'Tree' has no 'oid' member (no-member)
gitrevise/odb.py:687:24: E1101: Instance of 'Blob' has no 'oid' member (no-member)
gitrevise/odb.py:687:40: E1101: Instance of 'Blob' has no 'body' member (no-member)
************* Module gitrevise.tui
gitrevise/tui.py:214:8: R1722: Consider using sys.exit() (consider-using-sys-exit)
gitrevise/tui.py:217:8: R1722: Consider using sys.exit() (consider-using-sys-exit)
gitrevise/tui.py:220:8: R1722: Consider using sys.exit() (consider-using-sys-exit)
gitrevise/tui.py:223:8: R1722: Consider using sys.exit() (consider-using-sys-exit)
************* Module gitrevise.merge
gitrevise/merge.py:139:12: E1101: Instance of 'Blob' has no 'oid' member (no-member)
```

**Comment by gerboengels** [comment]
Of the 3 reported issues, two appear to be false positives and are issues of pylint itself:
`class-variable-slots-conflict`: https://github.com/PyCQA/pylint/issues/3141
`no-member`: https://github.com/PyCQA/pylint/issues/3143

**Comment by mystor** [comment]
Thanks! Should be fixed in f307b0eb8bae8066a0b8fd44691e5c4ce718e323, where I've also pinned the pylint version so this doesn't happen in the future.
